templates/wizards/projects/asn1acn/updateSourcesList.pri

```diff
@@ -105,10 +105,10 @@ names = $$createFileNames($$DISTFILES)
 generateC {
     cNames = $$names
     !isEmpty(names) {
-        cNames += asn1crt real
+        cNames += asn1crt
 
         contains(ASN1SCC_C_OPTIONS, --acn-enc) {
-            cNames += acn
+            cNames += asn1crt_encoding asn1crt_encoding_acn asn1crt_encoding_uper
```

Comment:
why uper when ACN was selected?

what is in the "encoding" (without any suffixes)? shouldn't it be added "always"?

Reply:
done